packages/jest-cli/src/getChangedFilesPromise.js

```diff
@@ -29,10 +29,10 @@ export default (
     }).catch(e => {
       const message = formatExecError(e, configs[0], {noStackTrace: true})
         .split('\n')
-        .filter(line => !line || !line.includes('Command failed:'))
-        .join('\n');
+        .filter(line => !!line && !line.includes('Command failed:'))
```

Comment:
As mentioned in the issue, this line should be just `.filter(line => !line.includes('Command failed:'))`. `line` is always a `string`, so it's essentially dead code 🙂 The check was if `line` was `null` or `undefined`, not empty.

Then you can remove the other changes, the linefeeds will be correct.